packages/mui-joy/src/List/List.tsx

```diff
@@ -46,7 +46,7 @@ export const StyledList = styled('ul')<{ ownerState: ListOwnerState }>(({ theme,
         '--ListDivider-gap': '0.25rem',
         '--ListItem-minHeight': '2rem',
         '--ListItem-paddingY': '3px',
-        '--ListItem-paddingX': '0.5rem',
+        '--ListItem-paddingX': ownerState.marker ? '3px' : '0.5rem',
```

Comment:
I'm curious about using a pixel value here, whereas all the other sizes use rem. 🤔

Reply:
When the value is smaller than `0.25rem`, it could result in a decimal `px`. However, no hard preference on this.

Reply:
Sounds good! Just wanted to understand — no need to change 🤙